backend/app/model/frontend/Chip.scala

```diff
@@ -46,14 +53,29 @@ object Chip {
   }
 }
 
+case class ParsedChips (query: String, workspace: Option[WorkspaceSearchContextParams])
+
 object Chips {
   // TODO - when the index supports attempts we can uncomment these lines to make this attempty
   //def parseQueryString(q: String): Attempt[String] = {
-  def parseQueryString(q: String): String = {
+  def parseQueryString(q: String): ParsedChips = {
     //Attempt.catchNonFatal {
+    val parsedQ = Json.parse(q)
+    // find WorkspaceSearchContextParams
```

Comment:
it might be worth adding a slightly more detailed comment here explaning what's happening - i.e extracting the workspace parameters, then ignoring them in the next bit